<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Assignment_10_11Class</class>
 <widget class="QMainWindow" name="Assignment_10_11Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assignment_10_11</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="coatsListWidget"/>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sizeLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Colour</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="colourLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Link</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="photoLineEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="givenSizeLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Given Size</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="priceButton">
      <property name="text">
       <string>Show Price</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalPriceLabel">
      <property name="text">
       <string>Total Price of Size:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Assignment_10_11.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>Assignment_10_11Class</receiver>
   <slot>addCoat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>565</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>634</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>priceButton</sender>
   <signal>clicked()</signal>
   <receiver>Assignment_10_11Class</receiver>
   <slot>showTotalPrice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCoat()</slot>
  <slot>showTotalPrice()</slot>
 </slots>
</ui>
